executor/syscalls_fuchsia.h

```diff
@@ -17,17 +17,14 @@ call_t syscalls[] = {
     {"dup2", 0, (syscall_t)dup2},
     {"dup3", 0, (syscall_t)dup3},
     {"faccessat", 0, (syscall_t)faccessat},
-    {"fchdir", 0, (syscall_t)fchdir},
```

Comment:
This patch seems to miss changes to sys/fuchsia/posix.txt that remove these syscalls.
But why are we removing them?